### PR TITLE
docs: move minimum reproduction template from wiki to repo

### PR DESCRIPTION
**Pull request by dundargoc**
Instead of a markdown file, just use a lua file instead that users can
use directly.

### CODE REVIEW

File: contrib/minimal.lua [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  -- some_plugin = 'https://github.com/author/plugin.nvim'
+} do
+  local install_path = vim.fn.fnamemodify('nvim_issue/'..name, ':p')
+  if vim.fn.isdirectory(install_path) == 0 then
+    vim.fn.system { 'git', 'clone', '--depth=1', url, install_path }
+  end
+  vim.opt.runtimepath:append(install_path)
+end
+
+-- ADD INIT.LUA SETTINGS THAT IS _NECESSARY_ FOR REPRODUCING THE ISSUE
```
Comment (clason) [inline contrib/minimal.lua:14]:
```suggestion
-- ADD INIT.LUA SETTINGS _NECESSARY_ FOR REPRODUCING THE ISSUE
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ body:
       description: |
         - For build failures: list the exact steps including CMake flags (if any).
         - For startup or shell-related problems: try `env -i TERM=ansi-256color "$(which nvim)"`.
-        - See [Minimal-reproduction-template](https://github.com/neovim/neovim/wiki/Minimal-reproduction-template) for how to create a minimal configuration.
+        - Use the minimal reproduction template provided in `contrib/minimal.lua` to create a minimal configuration. After you fill it out with necessary information, run with `nvim --clean -u contrib/minimal.lua`.
```
Comment (clason) [inline .github/ISSUE_TEMPLATE/bug_report.yml:28]:
```suggestion
        - Use the provided [minimal reproduction template](https://github.com/neovim/neovim/blob/master/contrib/minimal.lua) to create a minimal configuration. After you fill it out with necessary information, run with `nvim --clean -u minimal.lua`.
```

**Review by clason — APPROVED** [review]

**Comment by clason** [comment]
Also add a link to https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md#reporting-problems?

**Comment by dundargoc** [comment]
Done. Check again.

**Review by clason — APPROVED** [review]

**Pull request merged**